### ISSUE TITLE
Computer Vision Projects and Others

### ISSUE THREAD

**Issue by fxx40a3** [fxx40a3/Research#20]
https://www.analyticsvidhya.com/blog/2022/06/deploying-a-keras-flower-classification-model/										Not OK
										
Language Translator										
https://github.com/leeqv/kana-board/blob/master/media/insert-kanji.gif										
										
https://www.geeksforgeeks.org/python-classifying-handwritten-digits-with-tensorflow/										
										
Course/Research/Study										
	https://www.geeksforgeeks.org/courses/full-stack-applied-data-science-program?utm_campaign=554_abt&utm_medium=gfgcontent_abt&utm_source=geeksforgeeks									
Visualizing Convolutional Neural Networks | Layer by Layer										
	https://www.youtube.com/watch?v=JboZfxUjLSk&ab_channel=far1din									
										
Blur image detection using SVM										
	https://www.kaggle.com/code/norod78/blur-image-detection-using-svm/comments									
	OK									
Blur image detection using SVM										
	Code	https://www.kaggle.com/code/norod78/blur-image-detection-using-svm/notebook								
	Dataset	https://www.kaggle.com/datasets/kwentar/blur-dataset								
	OK									
	Code	https://www.kaggle.com/code/valentinld/blurdetection/notebook								
	Dataset	https://www.kaggle.com/datasets/kwentar/blur-dataset								
		/kaggle/input/blur-dataset/blur_dataset_scaled/								
		OK								
	Code	https://github.com/avs-abhishek123/Computer-Vision-Projects/tree/main/Contour_detection								
		OK								
	Code	https://www.kaggle.com/code/deshwalmahesh/image-processing-2-contour-edge-threshold								
		OK								
										
										
	Code	https://www.kaggle.com/code/dev523/leaf-disease-detection/notebook								
	Dataset	https://www.kaggle.com/code/dev523/leaf-disease-detection/input								
		OK								
	Code	https://www.kaggle.com/code/divyanshurajput75/leaf-disease-detection/notebook								
	Dataset	https://www.kaggle.com/code/dev523/leaf-disease-detection/input								
		working on								
	Code	https://www.kaggle.com/code/eyadgk/leafdiseasepred-imageclassificationtemp-densenet/notebook								
	Dataset	https://www.kaggle.com/code/dev523/leaf-disease-detection/input								
		working on								
										
										
100 Computer Vision Projects for 2023										
----------------------------------------										
# https://www.youtube.com/watch?v=Q16PMyIIE3Y&ab_channel=Murtaza%27sWorkshop-RoboticsandAI										
										
										
00:57 Object Size Measurement   										
01:22 Document Scanner  										
01:39 Traffic Sign Detection    										
02:22 Face Tracking with 2-axis PID Control 										
02:49 Face Door Lock    										
03:31 Face Emotion Detection    										
03:44 Face Detection    										
03:55 Shape Detection   										
04:26 Custom Object Detection   										
04:42 Obstacle Avoiding Robot   										
05:03 Optical Mark Recognition  										
										
05:42 Object Tracking   										
06:17 QR Code Detection 										
06:52 Text Detection    										
07:18 Face Filter   										
07:50 Feature Detection 										
08:16 Augmented Reality with Aruco Markers  										
08:47 Self-Driving Simulation   										
09:09 Angle Finder  										
09:29 Panorama Sticking 										
10:12 OCR in Forms  										
10:43 Self-Driving a Car using Neural Networks with Raspberry Pi    										
11:12 Sudoku Solver 										
11:40 Deep Fake 										
11:58 Drone Path Visualiser 										
12:16 Lane Following Drone  										
12:14 Eye Trakcing with Jetson Nano 										
13:47 Lane Following for self-driving car   										
14:27 Object Tracking with Jetson Nano Robot    										
15:00 Augmented Reality 										
15:35 Finger Counter    										
15:59 Track and Shoot using Computer Vision 										
16:18 Object Classification 										
16:42 Drone Face Following  										
17:00 Background Removal    										
17:21 Selfie Drone  										
18:17 Drone Flip using Hand Gestures    										
19:01 Drone Object Following    										
19:37 Ai Virtual Keyboard   										
19:59 Multiple Hand Gesture Control 										
20:20 Virtual Zoom Gesture  										
20:43 Hand Pose Detection   										
20:57 Virtual Quiz Game 										
21:21 Robot Hand Gesture Controlled 										
21:52 Custom Object Counter on Conveyor Belt    										
22:31 Cookie Cutter Squid Game  										
22:59 Green Light Red Light Squid Game  										
23:15 Automated Velcro Dart Board   										
23:29 Ballon Pop Game   										
23:52 Hand Distance Measurement 										
24:32 Virtual Paint 										
24:46 Fruit Classifier  										
25:01 Color Detection   										
25:46 Face Mesh 										
25:59 Body Pose Detection   										
26:17 MCQ Automated Grading 										
26:34 Virtual Gym   										
26:53 Drowsiness Detection  										
27:07 ID card Recognition   										
27:17 PPE Detection 										
27:34 Pistol Detection  										
27:56 Virtual Drag and Drop 										
28:10 Text Extraction on the web    										
28:25 Real-time Digit Classification    										
28:49 Virtual Calculator    										
29:21 Face Mask Detection   										
29:34 Kids Educational Game using Computer Vision   										
30:02 Waste Classifier  										
30:30 Glasses Try On    										
30:49 AI Robot Arm  										
31:23 Blink Counter 										
31:32 Face Distance Measurement 										
31:49 3d Motion Capture 										
32:06 Computer Vision Snake Game    										
32:39 3d Ball Tracking  										
33:35 Hand Gesture Fruit Ninja  										
34:01 Pong Game using Hand Gestures 										
34:42 AI Typing Tutor   										
35:20 Rotate 3D Objects with Your Head  										
36:06 Rock Paper Scissor    										
37:03 Hand Sign Detection   										
37:30 Object Detection on the web   										
37:45 AI Virtual Mouse  										
38:06 Car Counter   										
38:17 3d Hand Tracking  										
39:02 Suspicious luggage    										
39:12 Coffee Machine Virtual Interface  										
39:49 Computer Vision Pop Game  										
40:08 Basketball Shot Predictor 										
40:34 Gesture Controlled Presentation   										
40:52 Intruder Detection    										
41:07Face Recognition with Real-time Database   										
41:34 Customer Engagement   										
41:56 Shop traffic counter  										
42:10 Drone Body Following  										
42:43 License Plate Recognition 										
42:59 Gesture Volume Control    										
43:15 Shirt Size Measurement    										
# https://www.datacamp.com/blog/computer-vision-projects?utm_source=google&utm_medium=paid_search&utm_campaignid=19589720821&utm_adgroupid=152984011814&utm_device=c&utm_keyword=&utm_matchtype=&utm_network=g&utm_adpostion=&utm_creative=714994855825&utm_targetid=aud-1459190388940:dsa-2222697810918&utm_loc_interest_ms=&utm_loc_physical_ms=9197070&utm_content=DSA~blog~Machine-Learning&utm_campaign=230119_1-sea~dsa~tofu_2-b2c_3-row-p1_4-prc_5-na_6-na_7-le_8-pdsh-go_9-nb-e_10-na_11-na-oct24odp&gad_source=1&gclid=Cj0KCQjw6oi4BhD1ARIsAL6pox3jGI1p_TmDtJjepgdnI6mC9BbWazyehEP_mrCab2WtIIRcWMIF4dMaAuEAEALw_wcB										
Optical Character Recognition (OCR) for Handwritten Text										
Plant Disease Detection										
Honey Bee Detection										
Clothing Classifier										
Food Image Classification										
Multi-object Tracking in Video										
Image Captioning										
3D Object Reconstruction From Multiple Views										
Gesture Recognition For Human-Computer Interaction										
Visual Question Answering (VQA)										
Insurance Code Extraction										
Image Deblurring										
Video Summarization										
Face De-Aging/Aging										
Human Pose Estimation And Action Recognition in Crowded Scenes										
Unsupervised Anomaly Detection in Industrial Inspection										
										
										
# https://www.geeksforgeeks.org/computer-vision-projects/										
										
Detect the RGB color from a webcam using Python – OpenCV										
Face Detection using Python and OpenCV with a webcam										
Face and Hand Landmarks Detection using Python – Mediapipe, OpenCV										
Real-Time Edge Detection using OpenCV										
Implement Canny Edge Detector in Python using OpenCV										
Gun Detection using Python-OpenCV										
Real-time object color detection using OpenCV										
Right and Left Hand Detection Using Python										
Age Detection Using Deep Learning in OpenCV										
OpenCV – Drowsiness Detection										
Build GUI Application Pencil Sketch from Photo in Python										
Measure Size of an Object Using Python OpenCV										
Brightness Control With Hand Detection using OpenCV in Python										
Car driving using hand detection in Python										
Contour Detection with Custom Seeds using Python – OpenCV										
Find Co-ordinates of Contours using OpenCV | Python										
Live Webcam Drawing using OpenCV										
Black and white image colorization with OpenCV and Deep Learning										
Detect and Recognize Car License Plate from a video in real-time										
License Plate Recognition with OpenCV and Tesseract OCR										
Handwritten Digit Recognition using Neural Network										
Image Classification with Convolutional Neural Networks (CNNs) Using Pytorch										
Human Pose Estimation with OpenCVUsing										
Age and Gender Prediction from Image Using Pytorch										
Face detection using Cascade Classifier using OpenCV-Python										
Face recognition using Artificial Intelligence										
Face recognition using GUI										
FaceMask Detection using TensorFlow in Python										
Python OpenCV – Super-resolution with deep learning										
Real-Time Road Lane Detection										
CIFAR-10 Image Classification in TensorFlow										
Dog Breed Classification Using Transfer Learning										
Flower Recognition Using Convolutional Neural Network										
Emojify using Face Recognition with Machine Learning										
Cat & Dog Classification using Convolutional Neural Network in Python										
Video Analysis with Convolutional LSTM Networks										
Autonomous Driving with Deep Learning										
Visual Question Answering with Neural Networks										
Scene Recognition with Convolutional Neural Networks										
Text Recognition from Image with Convolutional Recurrent Neural Networks										
Image Captioning Using Pytorch										
Medical Image Analysis Using Deep Learning										
Document Layout Analysis with Deep Learning										
Deep Learning for Artistic Style Transfer										
Landmark Recognition with Deep Neural Networks										
Image Restoration with Generative Adversarial Networks										
Satellite Image Analysis Using Convolutional Neural Network										
Image generation with Convolutional Autoencoders										
Lung Cancer Detection using Convolutional Neural Network (CNN) Tensorflow										
Lung Cancer Detection Using Transfer Learning										
Lung cancer detection using PyTorch										
Pneumonia Detection Using Deep Learning										
Detecting Covid-19 with Chest X-ray										
Skin Cancer Detection Using TensorFlow										
Hand Gesture Recognition with Convolutional Neural Networks										
Optical Character Recognition Using Pytorch										
Video Object Segmentation with Convolutional Networks										
Traffic Signs Recognition using CNN and Keras in Python										
Vehicle license plate recognition										
Image Inpainting with Deep Neural Networks										
Image Deblurring with Deep Neural Networks										
Object Detection Using YOLO Using Pytorch										
Object Detection Using YOLO Using Tensorflow										
Object Detection with Faster R-CNN and Mask R-CNN										
Image Segmentation with U-Net using Tensorflow										
Image Segmentation with SegNet using Tensorflow										
Image Segmentation with U-Net using Pytorch										
Neural Style Transfer Using Pytorch										
Image Synthesis using Generative Adversarial Networks (GANs)										
Emotion Recognition through Facial Expressions										
Facial expression detection using the Deepface module in Python										
Instance segmentation using Pytorch										
Semantic segmentation using Pytorch										
Face similarity Detection using PyTorch										
Age Conversion Using GANs Algorithm										
Business Card Recognization Using Deep Learning										
Traffic light detection using Deep Learning										
People Counting Using Deep Learning										
DeepDream using CNNs										
Cartoonifying an Image Using Deep Learning										
Road Condition Monitoring Project Using Deep Learning										
Image colourization using GANs algorithm										
Image Classifier using Tensorflow Object Detection API										
										
Excellant Repository			https://github.com/avs-abhishek123/Computer-Vision-Projects							
500-AI-Machine-learning-Deep-learning-Computer-vision-NLP-Projects-with-code			https://github.com/ashishpatel26/500-AI-Machine-learning-Deep-learning-Computer-vision-NLP-Projects-with-code							
										
https://machinelearningprojects.net/computer-vision-projects/										
										
Background-Remover-Flask-App										
Leaf Disease Detection Flask App										
Drowsiness Detection using cv2										
Number Plate Detection using Yolov7										
Train yolov7 on the custom dataset										
Blur Faces in Live Feed										
Sketch making Flask App										
Generating cifar 10 fake images using DCGAN										
Helmet and Number Plate Detection and Recognition using YOLOv3										
healthcure an all in one medical solution										
Invisible Man using Mask RCNN										
Sudoku Solver										
Object Detection using SSD										
Social Distancing using YOLOv3										
Face Recognition based Attendance System										
detect shapes using cv2										
Document Scanner using OpenCV										
Face Landmarks Detection using dlib										
Make your Sketch using OpenCV										
How to perform Face Recognition using KNN										
Immortal Snake game in Python										
find the most dominant colors in an image using KMeans clustering										
types of thresholding techniques										
Face and eye detection in cv2										
Harrys Invisibility Cloak										
split and merge channels in cv2										
edge detection using Sobel X and Sobel Y										
Rotating and Scaling Images in cv2										
use mouse clicks to draw circles in Python										
repair damaged images using inpainting methods										
generate negative images in Python										
detect edges using Laplacian 2nd order derivative										
plot a Histogram of a grayscale image 1										
denoise an Image using Median Blur										
perform Morphological Operations like Erosion Dilation and Gradient 1										
detect contours in an Image										
plot Color Channels Histogram of an Image										
Blurrings in cv2										
BGR Palette in OpenCV										
draw basic shapes on Images										
resize an image in Python										
read an image in python										
										
# https://neptune.ai/blog/15-computer-visions-projects										
										
1. Edge & Contour Detection 										
2. Colour Detection & Invisibility Cloak										
3. Text Recognition using OpenCV and Tesseract (OCR)										
4. Face Recognition with Python and OpenCV										
5. Object Detection										
6. Hand Gesture Recognition										
7. Human Pose Detection										
8. Road Lane Detection in Autonomous Vehicles										
9. Pathology Classification										
10. Fashion MNIST for Image Classification										
11. Image Deblurring using Generative Adversarial Networks										
12. Image Transformation 										
13. Automatic Colorization of Photos using Deep Neural Networks										
14. Vehicle Counting and Classification										
15. Vehicle license plate scanners										
										
# https://www.youtube.com/watch?v=BFlRmIvqwSA&t=291s&ab_channel=Murtaza%27sWorkshop-RoboticsandAI										
Background Removal:    • Background Removal Like Zoom | OpenCV...  										
Virtual Calculator:    • Virtual Calculator using OpenCV Pytho...   										
Velcro Dart Game:    • Computer Vision Game using OpenCV Pyt...  										
Red Light Green Light (Paid): https://www.computervision.zone/cv-ga...										
Cookie Cutter(Paid): https://www.computervision.zone/cv-ga...										
Personal AI Trainer:    • Build a Personal AI Trainer | OpenCV ...  										
Track and Shoot:    • Track and Shoot using Computer Vision...  										
Elbow pose detector (Paid): https://www.computervision.zone/ardui...										
AI Mouse Control:    • AI Virtual Mouse | OpenCV Python | Co...  										
Custom Object Counter on Conveyor belt (Paid): https://www.computervision.zone/ardui...										
Robot Hand Gesture Arduino (Paid): https://www.computervision.zone/ardui...										
Virtual Keyboard:    • AI Virtual Keyboard using OpenCV | Co...  										
Self Driving (MegaBot):    • Is this the best Raspberry Pi Robotic...  										
AI Virtual Painter:    • AI Virtual Painter | OpenCV Python | ...  										
Balloon Pop Pygame (Paid): https://www.computervision.zone/cv-ga...										
Object Detection Mobile APP (Paid): https://www.computervision.zone/cv-mo...										
Face Tracking (Paid): https://www.computervision.zone/ardui...										
Eye Tracking (Paid): https://www.computervision.zone/ardui...										
Deep Fake:    • Deepfake Tutorial and Explanation Ste...   										
Balloon Pop Game:    • I tried building a Computer Vision Ga...  										
Basketball Shot Predictor:    • Basket Ball Shot Predictor using Open...  										
Hand Distance measurement:    • Hand Distance Measurement with Normal...   										
Parking Space Counter:    • Parking Space Counter using OpenCV Py...  										
Volume Gesture Control:    • Gesture Volume Control | OpenCV Pytho...  										
Drone Body Following (Paid): https://www.computervision.zone/adv-d... 										
										
										
https://www.projectpro.io/article/object-detection-project-ideas-with-source-code/490										
										
4) Monkey, Cat, and Dog Detection System										
6) Train your Own Object Detection Model										
9) Vehicle Counting Model										
10)  Furniture Recognition Model										
11) Vehicle Detection in Video Data										
12) Surveillance Camera Object Detection System										
14) Build an Object Detection Web Application										
15) Motion Detection System
